src/hierarchies.rs

```diff
@@ -31,6 +32,72 @@ use crate::{Controllers, Hierarchy, Subsystem};
 
 use crate::cgroup::Cgroup;
 
+/// Process mounts information.
+///
+/// See `proc(5)` for format details.
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct Mountinfo {
+    // /// Mount pathname relative to the process's root.
```

Comment:
I think `//`before `///` should be removed.

Reply:
done